.github/workflows/preview-env-delete.yml

```diff
@@ -1,4 +1,4 @@
-name: "Delete preview environment"
+name: "Preview environment delete"
```

Comment:
Just changed this name so that the "Preview environment X" jobs are grouped together due to sorting in the list of workflows.